toolkit/tools/imagecustomizer/docs/iso.md

```diff
@@ -111,15 +105,20 @@ os:
 ### Full Disk Image
 
 The input full disk image must satisfy the following requirements in order for
-MIC to be able to generate an iso image out of it:
+the Azure Linux Image Cuztomizer to be able to generate an iso image out of it:
 - File layout (after all partitions have been mounted):
   - `/boot/grub2/grub.cfg` must exist and is the 'main' grub configuration (not
     a redirection grub configuration file for example).
   - The bootloader and the shim must exist under the `/boot` folder or any of
     its subdirectories.
     - For x64, `bootx64.efi` and `grubx64.efi` (or `grubx64-noprefix.efi`).
-    - For ARM64, `bootaa64.efi` and `grubaa64.efi` (or `grubx64-noprefix.efi`).
+    - For ARM64, `bootaa64.efi` and `grubaa64.efi` (or `grubaa64-noprefix.efi`).
   - All grub configurations and related files must be stored under the `/boot`
     folder. For example, grub.cfg cannot reference files outside that folder.
     If it does, those referenced files will not be copied to the iso and may
     cause grub to fail booting the desired operating system.
+
+### LiveOS ISO Image
+
+The input LiveOS iso image must be previously generated by The Azure Linux Image
```

Comment:
nit - lower case `The` or remove it completely.

Reply:
pending.